soc/nxp/imx/imx9/imx95/Kconfig

```diff
@@ -10,6 +10,8 @@ config SOC_MIMX9596_M7
 	select CPU_HAS_DCACHE
 	select CPU_HAS_ARM_MPU
 	select ARM_MPU
+	select CACHE_MANAGEMENT
```

Comment:
Don't select it directly, set default value of "y" in Kconfig.defconfig in case of someone want to disable it.